docs/user-guide/configmap.md

```diff
@@ -518,6 +518,7 @@ The value can either be:
 - round_robin: to use the default round robin loadbalancer
 - least_conn: to use the least connected method
 - ip_hash: to use a hash of the server for routing.
+- ewma: to use the peak ewma method for routing 
```

Comment:
Currently this LB algorithm is available only when dynamic configuration is enabled, we should somehow convey that info here.